packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodeRow.tsx

```diff
@@ -100,7 +101,7 @@ export const NodeRow = React.memo((props: NodeRowProps) => {
           </>
         )}
       </TableCell>
-      <TableCell>
+      <TableCell noWrap={true}>
```

Comment:
Wow I didn't even know there was a `noWrap` prop! You can probably make it even more terse:

```suggestion
      <TableCell noWrap>
```

Reply:
👍